projects/core/components/dialog/dialog.tokens.ts

```diff
@@ -0,0 +1,29 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+import {TuiDialogOptions} from '@taiga-ui/core/interfaces';
+
+export type TuiDialogDefaultOptions = Omit<TuiDialogOptions<unknown>, 'data'>;
+
+export const TUI_DIALOG_DEFAULT_OPTIONS: TuiDialogDefaultOptions = {
+    size: `m`,
+    required: false,
+    closeable: true,
+    dismissible: true,
+    label: ``,
+    header: ``,
+};
+
+export const TUI_DIALOG_OPTIONS = new InjectionToken<TuiDialogDefaultOptions>(
+    `Default parameters for dialog component`,
```

Comment:
We have new eslint rule for injection tokens's description.

Rebase is required :(